### PR TITLE
ENG-122: fix element sizes

### PR DESCRIPTION
**Pull request by afolasope**

### CODE REVIEW

File: apps/frontend/src/modules/surveys/components/studio/distribute/sharable-links/QRCodeview.tsx [hunk 1]
```diff
@@ -48,12 +48,12 @@ export default function QRCodeView({ url, name }: { url: string; name: string })
     };
 
     return (
-        <div className="flex w-[778px] flex-col items-center gap-8 pt-8">
+        <div className="flex w-[600px] flex-col items-center gap-8 pt-8">
```
Comment (AfeezGL) [inline apps/frontend/src/modules/surveys/components/studio/distribute/sharable-links/QRCodeview.tsx:51]: @afolasope I thing this width should be based on screen size.

Comment (AfeezGL) [reply apps/frontend/src/modules/surveys/components/studio/distribute/sharable-links/QRCodeview.tsx:51]: So, for this container, we can add `lg:w-[778px]` to the `classNames` and for the `QRCode` component, we can can add `size-[210px] lg:size-[397px]` to the `classNames`

File: apps/frontend/src/modules/workspace/components/settings/Workspace/Member.tsx [hunk 2]
```diff
@@ -43,23 +43,23 @@ export const Member = () => {
...
 
                 <div className="flex w-full items-end gap-2">
-                    <div className="w-[75%]">
+                    <div className="flex-1">
                         <TextInput
                             placeholder=""
                             value={addEllipsis(inviteLink, 60)}
                             disabled={true}
                             rightIcon={
                                 <button disabled={loading} onClick={handleInviteLinkRefresh}>
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/settings/Workspace/Member.tsx:52]: @afolasope What is the point of the custom `Button` component we have if we still use the native component.

File: apps/frontend/src/modules/workspace/components/settings/Workspace/Member.tsx [hunk 3]
```diff
@@ -43,23 +43,23 @@ export const Member = () => {
...
                             }
                         />
                     </div>
-                    <div className="w-[25%]">
+                    <div>
                         <Button
                             text="Copy"
-                            className="text-sm"
-                            icon={<CopyIcon size={16} />}
+                            className=" h-[50px] w-[92px] p-3 text-sm"
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/settings/Workspace/Member.tsx:61]: @afolasope We should not be modifying button styles directly. It makes no sense to have a design system then we start sprinkling custom styles across the codebase, it makes it very difficult to maintain. What we should rather have is the concept of `variant` like `sm`, `lg` `xl` e.t.c.

File: apps/frontend/src/modules/workspace/components/settings/Workspace/Member.tsx [hunk 4]
```diff
@@ -86,7 +86,11 @@ export const Member = () => {
                         />
                     </div>
                     <div>
-                        <Button className="flex items-center gap-2 " size="md" onClick={() => setOpenInviteModal(true)}>
+                        <Button
+                            className="flex items-center gap-2 text-sm"
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/settings/Workspace/Member.tsx:90]: @afolasope The same here too.

File: apps/frontend/src/modules/workspace/components/invite/WorkspaceInvite.tsx [hunk 5]
```diff
@@ -124,16 +124,12 @@ export const WorkspaceInvite = ({ open, onOpenChange }: Props) => {
...
-        return <GlobalSpinner />;
-    }
-
     return (
         <Dialog open={open} onOpenChange={onOpenChange}>
             <DialogContent
                 alignHeader="left"
                 title={`Invite others to '${workspace?.name}'`}
-                description="Invite others to your project to collaborate together in intergraflow. An invite is specific to an email address and expires after 3 days."
+                description="Invite others to your project to collaborate together in integraflow. An invite is specific to an email address and expires after 3 days."
```
Comment (jdamilola) [inline apps/frontend/src/modules/workspace/components/invite/WorkspaceInvite.tsx:132]: @afolasope Can you check all occurrences of `integraflow` and then change them to `Integraflow`.

**Review by jdamilola — APPROVED** [review]

**Comment by jdamilola** [comment]
@afolasope I'll merge this PR once you address my comment.

**Review by jdamilola — APPROVED** [review]

**Pull request merged**